Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -518,22 +518,29 @@ void HexEditorWidget::update_inspector_values(size_t position)
 
     auto selected_bytes = m_editor->get_selected_bytes();
 
-    auto ascii_string = String::from_utf8(ReadonlyBytes { selected_bytes });
+    // Replace control chars with something else - we don't want line breaks here ;)
+    auto replace_control_chars = [](ErrorOr<String> string_or_error) {
+        if (string_or_error.is_error())
+            return string_or_error;
+        return string_or_error.release_value().replace("\n"sv, " "sv, ReplaceMode::All);
```

Comment:
Should this replace all chars < 32 with \xAB (or the well known ones with \n \t \r etc)? Space seems a bit lossy, maybe we could do something more useful:)

Reply:
Yeah, I meant to leave a TODO for that but seem to have forgotten. I tried not doing anything too fancy because there are a lot of questions, do we only replace *some* characters, and what do we replace with, etc. so I just did the simple, dumb thing that fixes the most annoying issue.